core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/python/PythonUDFOpDescV2.scala

```diff
@@ -27,25 +27,28 @@ class PythonUDFOpDescV2 extends OperatorDescriptor with PortDescriptor {
     defaultValue =
       "# Choose from the following templates:\n" +
         "# \n" +
-        "# from pytexera import *\n" +
+        "# import pytexera as pt\n" +
+        "# from overrides import overrides\n" +
+        "# from typing import Iterator, Optional\n" +
         "# \n" +
-        "# class ProcessTupleOperator(UDFOperatorV2):\n" +
+        "# \n" +
```

Comment:
adding a new line to fix the warning of single blank line between imports and class definitions.